docs/monthly-meeting/2022-06.md

```diff
@@ -0,0 +1,175 @@
+
+Petr: What does this mean in practice?
+
+Guido: Being in charge of the agenda, how long things are discussed.
+
+Petr: Fine with that, that's how things work today. Does this need to be formalised somehow?
+
+Guido: Someone should have 'Convening' power -- someone to take ownership of filling the agenda, managing the admin, etc.
+
+Adam: Typing has more active solicitation of agenda items. That might be a concrete improvement/action item out of this.
+
+Petr: *in agreement*
+
+CAM: The Documentation category on Discourse serves this role, and we have posts on there for some of these meetings.
+
+Ned: Where are decisions made? Typing-sig makes decisions affecting them, as it is a forum of typing-checker implementers and users. Can Discourse be a place decisions are made?
+
+Ned: There's agreement about *that* we should use Diataxis, but not about *how* we should implement it.
+
+- [ ] Petr: Solicit agenda items on Discourse, for the next/future meetings
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, &c.*
+
+#### Proofreaders GitHub team
+
+It would be useful to have a GitHub team that can be `@mentioned` in docs PRs to request reviews of the English grammar.
+
+- [ ] Petr: Create a `proofreaders` team (Initially: CAM, Adam, Pradyun, Hugo), post about this team on Discourse (or similar) for a wider audience
+
+#### Devguide improvement plans
+
+Ezio made some comments in [a thread](https://discord.com/channels/935215565872693329/980818577559007242):
+
+> The first plan is to overhaul a bit the GitHub docs, and is also related to other changes about labels.
+The second plan is still somewhat half-baked, but I wanted to
+> 1. add some instructions about following the development of CPython
+
+Ezio: Add a changelog page to the devguide for an overview of workflow changes
+
+- [ ] Ezio: Add a “what's new” section to the devguide (key/important changes)
+
+CAM: Two different types of content: new pages on tools and workflow versus changes to existing policies and proceduces.
+
+Guido: Not that useful to distinguish these, as it adds more categories. Everything is new.
+
+Ned: Is a problem that we have different audiences? E.g. triagers versus developers versus contributors to Python
+
+Guido: (core devs & triagers), contributors, redistributors/people who need to build Python.
+
+> 2. add a "tips and tricks" page
+> There is some overlapping between https://devguide.python.org/communication/, the instructions that I want to add, the tips and tricks page, and other pages like the GitHub bootcamp though, so I'm still trying to figure out all the things I want to include and then what would be the best way to organize them.
+
+Ned: This sounds like Ezio is suggestion that the devguide should have a 'how to' section (one of the four sections of Diataxis).
+
+- [ ] Ezio: Add a (How-To) Guides section with these
```

Comment:
I will create an issue/project later.  I still need more time to think what would be the best way to approach this.